src/pudl/metadata/resources/__init__.py

```diff
@@ -201,8 +201,84 @@
     },
     "boilers": {
         "id_cols": ["plant_id_eia", "boiler_id"],
-        "static_cols": ["prime_mover_code"],
-        "annual_cols": [],
+        "static_cols": [
+            "prime_mover_code",
+            "state",
+            "county",
+            "operating_date",
+            "retirement_date",
```

Comment:
I think all of these column names appear in other tables too, which will probably break the harvesting process. I think each attribute is expected to be uniquely associated with a single entity. The `state` and `county` (and other location fields) are part of `plants_entity_eia`, while `prime_mover_code` is associated with `generators_eia860` I could see there being separate operating & retirement dates for generators and boilers, but in that case they'll need to have unique names, e.g. `generator_operating_date` vs. `boiler_operating_date`, otherwise they'll be confused by the harvesting process.

Reply:
I know i'm jumping in later here but i'm not sure this is totally true about confusing the harvester. the harvesting would be confused if a table has **both** `boiler_id` and `generator_id` **and** these columns we want to harvest.

The harvester loops through these entities, finds the tables with the `id_cols` then rips all instances of the static and annual cols out of their original tables (unless those cols are in `keep_cols`). But from what I see of these new tables that your pulling in now, there aren't table with both `boiler_id` and `generator_id` in them. Although I could be missing it!

I'm personally partial to having the same column names for things even when they are referring to seperate entities if they generally mean the same thing.

Reply:
Jumping slightly ahead, when we integrate `emissions_control_equipment` we'll get another set of operating and retirement date columns, this time associated with the control equipment. This table will have the boiler ID in it and it's possible we'd want to aggregate some of the columns (e.g. total cost of control equipment) to add to the harvest (the rest have an m:1 relationship to boilers). Between this and the actual different meaning of "operating_date" between generators (actual operating date) and boilers (planned or actual), I wonder if distinguishing between source is the most useful here?

Reply:
yeah this is a good point. I think I am generally partial to have things named the same way but if it introduces lots of downstream confusion then I feel good about these more specific names.